ColorCoding.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+#include <assert.h>
+#include "ColorCoding.h"
+
+int main() {
+    ColorPair colorPair;
+    printf("PRINTING MANUAL\n");
```

Comment:
This function does the tests as well as production code (printing the manual).
This would make it impossible to deliver the printing of the manual, without delivering all the test code as well.
Also, we can't make out the purpose of the file from its name.